lib/version.js

```diff
@@ -25,20 +27,7 @@ function version (args, silent, cb_) {
   if (args.length > 1) return cb_(version.usage)
 
   var packagePath = path.join(npm.localPrefix, "package.json")
-  fs.readFile(packagePath, function (er, data) {
-    function cb (er) {
-      if (!er && !silent) console.log("v" + data.version)
-      cb_(er)
-    }
-
-    if (data) data = data.toString()
-    try {
-      data = JSON.parse(data)
-    }
-    catch (er) {
-      data = null
-    }
-
+  readJson(packagePath, function (er, data) {
```

Comment:
We don't want to use readJson here for a host of reasons*.  This should remain `fs.readFile + JSON.parse`.

\* Those reasons being:
* readJson can read package metadata from non-`package.json` sources (eg, `index.js` header comments)
* readJson passes the result through `normalize-package-data` which will mutate the json a variety of ways.
* readJson does a bunch of its own normalization
* readJson reads a bunch of standalone files and injects them into the resulting structure– AUTHORS, README*…
* And more… it does an awful lot.